config.json

```diff
@@ -0,0 +1 @@
+{"applicationId":160,"deviceType":"isg-503","userId":2,"appUpdatePollInterval":600000,"listenPort":48484,"vpnPort":443,"apiEndpoint":"https://api.cloud.nebra.com","vpnEndpoint":"vpn.cloud.nebra.com","registryEndpoint":"registry.cloud.nebra.com","deltaEndpoint":"https://delta.cloud.nebra.com","mixpanelToken":"__unused__","apiKey":"kpvyuEPPxOiJHHUIOBZrCJkzkp941aY7"}
```

Comment:
Don't think this is correct as wrong device type? Did you generate from the Balena cli?

Reply:
Nope, this is a copy from the `bobcat-px30`. I suggest that we stick with the same device type as it doesn't really matter as long as it's identified as a known one. Is there anything else that needs to be changed?

Reply:
I think the device will have to be different ... See slack message

Reply:
https://nebraltd.slack.com/archives/C024BNQ1Y6T/p1691097130419659